portfolio/src/main/webapp/script.js

```diff
@@ -1,22 +1,33 @@
 /**
  * Gets comments from the server.
  */
-async function getComments() {
-  fetch('/comments').then(response => response.json()).then((comments) => {
-    const historyEl = document.getElementById('comments-history');
-    comments.history.forEach((line) => {
-      historyEl.appendChild(createListElement(line));
+async function getComments(limit) {
+  const historyEl = document.getElementById('comments-history');
+  historyEl.innerHTML = ''; //clears the table of previous comments
+  fetch('/comments?comment-limit=' + limit).then(response => response.json()).then((comments) => {
+    comments.forEach((comment) => {
+      historyEl.appendChild(createListElement(comment));
     });
   });
 }
 
 /** Creates an <li> element containing text. */
-function createListElement(text) {
+function createListElement(comment) {
   const liElement = document.createElement('li');
-  liElement.innerText = text;
+  liElement.innerText = comment.text;
   return liElement;
 }
 
+/**
+ * Deletes comments from the server.
+ */
+async function deleteComments() {
+  const historyEl = document.getElementById('comments-history');
+  historyEl.innerHTML = ''; //clears the table of displayed previous comments
```

Comment:
I would put this line inside the response handler because it's possible that the delete may fail on the server side. If that happens, we don't want to remove the content on the page. In other words, remove the content on the page only when we get a successful response.

Reply:
Done.